src/app/FakeLib/MSBuildHelper.fs

```diff
@@ -446,7 +464,10 @@ let MSBuildWithProjectProperties outputPath (targets : string) (properties : (st
             |> Set.unionMany
 
     let setBuildParam project projectParams =
-        { projectParams with Targets = targets |> split ';' |> List.filter ((<>) ""); Properties = projectParams.Properties @ properties project }
+        { projectParams with 
+            Targets = targets |> split ';' |> List.filter ((<>) "")
+            Properties = projectParams.Properties @ properties project
+            MaxCpuCount = Some ( Some (EnvironmentHelper.getMachineEnvironment().ProcessorCount)) }
```

Comment:
@baronfel To be honest I don't think we should do such changes in FAKE 4 anymore. But I guess @forki probably has a different opinion on this. If it breaks somebody we will move this to FAKE 5 anyways :)